src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -289,15 +418,25 @@ def enable_local_interactive(resources, cluster_name, namespace):
     ][0].get("command")[2]
 
     command = command.replace("deployment-name", cluster_name)
-    try:
-        config_check()
-        api_client = client.CustomObjectsApi(api_config_handler())
-        ingress = api_client.get_cluster_custom_object(
-            "config.openshift.io", "v1", "ingresses", "cluster"
+
+    if is_openshift_cluster():
+        # We can try get the domain through checking ingresses.config.openshift.io
+        try:
+            config_check()
+            api_client = client.CustomObjectsApi(api_config_handler())
+            ingress = api_client.get_cluster_custom_object(
+                "config.openshift.io", "v1", "ingresses", "cluster"
+            )
```

Comment:
Same here. RE: These checks seem they are already performed as part of the `is_openshift_cluster()` func